spec.md

```diff
@@ -0,0 +1,461 @@
+# The UltraStar File Format
+
+The UltraStar file format is a timed text format for describing karaoke songs where singing performance is scored by the karaoke software.
+
+## Status of this document
+
+This document aims to describe the UltraStar File Format in its most recent version 2.0.0.
+
+> [!IMPORTANT]
+>
+> This document is currently a work-in-progress. Parts of the final specification may change significantly from the current state of this document.
+
+GitHub Issues are preferred for discussion of this specification. Alternatively, you can discuss comments on our Discord server.
+
+## 1. Introduction
+
+The UltraStar file format provides a standardized representation of karaoke songs. The format has been used for a long time by numerous karaoke games such as UltraStar Deluxe, Performous, or Vocaluxe. There exists an ecosystem of supporting applications for hosting, editing, and managing songs. However due to the lack of an official file format specification implementations differ and new features cannot be added in a consistent manner. This document aims to fix this problem by providing a formal specification for the syntax and semantics of the UltraStar file format.
```

Comment:
When reffering to other parties always include their GitHub link:
UltraStar Deluxe -> https://github.com/UltraStar-Deluxe/USDX
Performous -> https://github.com/performous/performous
Vocaluxe -> https://github.com/Vocaluxe/Vocaluxe

Reply:
Good point. Fixed.